application/controllers/admin/Tokens.php

```diff
@@ -1081,7 +1081,8 @@ public function addDummies(int $iSurveyId, string $subaction = '')
                 [
                     'showSaveAndCloseButton' => true,
```

Comment:
No way to renane "Save and close" to "Create" ?

Reply:
Text is not customizable. But could it be... let's leave it for another ticket

Reply:
Yep

> If no way : need to create an UX issue. It muts be “Create participants“